maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -63361,7 +63363,9 @@
 /turf/open/floor/almayer,
 /area/almayer/living/briefing)
 "tnb" = (
-/obj/structure/bed/chair/comfy/black,
+/obj/structure/bed/chair/comfy/black{
+	dir = 4
```

Comment:
Randomly rotated chair in port railgun/missile room.

Reply:
No idea why these rotated.